ProcessTree.cpp

```diff
@@ -553,8 +553,10 @@ std::string ProcessTree::ExtractContainerId(const std::string& exe, const std::s
     //containerd-shim -namespace moby 
     //-workdir /var/lib/containerd/io.containerd.runtime.v1.linux/moby/ebe83cd204c57dc745ce21b595e6aaabf805dc4046024e8eacb84633d2461ec1 
     //-address /run/containerd/containerd.sock -containerd-binary /usr/bin/containerd -runtime-root /var/run/docker/runtime-runc
+    // OR ...
+    // /usr/bin/containerd-shim-runc-v2 -namespace moby -id 6d8e2e01f3bb36fe420e0515066e1b6e0e44f7198e3fbc563a46396f75939b5e -address /run/containerd/containerd.sock
 
-    std::string containerid = "";
+    std::string containerid = "";   
     if (ends_with(exe, "/containerd-shim") && starts_with(cmdline, "containerd-shim -namespace moby")) {
```

Comment:
Please change `ends_with(exe, "/containerd-shim") && starts_with(cmdline, "containerd-shim -namespace moby")` to `ends_with(exe, "/containerd-shim") || ends_with(exe, "/docker-containerd-shim")`

Also, change the logic so that it grabs the last path component of the -workdir arg instead of assuming a fixed prefix.